Fixed #34384 -- Fixed session validation when rotation secret keys. (#16631)
django/django: docs/ref/contrib/auth.txt

    ``get_user()`` method to retrieve the user model instance and then verifies
    the session by calling the user model's
    :meth:`~django.contrib.auth.models.AbstractBaseUser.get_session_auth_hash`

>     method. If the verification fails and :setting:`SECRET_KEY_FALLBACKS` is
>     defined, it verifies the session against each fallback key using

Reviewer: I might just say `are provided` since SECRET_KEY_FALLBACKS defaults to `[]`, and so is likely `defined`

```suggestion
    method. If the verification fails and :setting:`SECRET_KEY_FALLBACKS` are
    provided, it verifies the session against each fallback key using
```

Reviewer 2: Updated :+1:
